packages/application-components/src/components/page-unauthorized/page-unauthorized.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import FailedAuthorizationSVG from '@commercetools-frontend/assets/images/folder-full-locked.svg';
+import MaintenancePageLayout from '../maintenance-page-layout';
+import { SUPPORT_PORTAL_URL } from '@commercetools-frontend/constants';
+import messages from './messages';
+
+// eslint-disable-next-line react/display-name
+const getSupportUrlLink = (msg: string) => (
+  <a href={SUPPORT_PORTAL_URL} target="_blank" rel="noopener noreferrer">
+    {msg}
+  </a>
+);
+
+export const PageUnauthorized = () => (
+  <MaintenancePageLayout
+    imageSrc={FailedAuthorizationSVG}
+    title={<FormattedMessage {...messages.title} />}
```

Comment:
Please add a `label` as recently added to other pages. It's just the title.

Reply:
Sorry, I don't quite get it, what label? This component doesn't seem to have such prop...

Reply:
It should: https://github.com/commercetools/merchant-center-application-kit/blob/master/packages/application-components/src/components/maintenance-page-layout/maintenance-page-layout.tsx#L54

Maybe your branch is not up-to-date with `master`?

Reply:
Yeah, my bad :(